lib/ansible/modules/network/nxos/nxos_linkagg.py

```diff
@@ -372,7 +372,7 @@ def main():
     """ main entry point for module execution
     """
     element_spec = dict(
-        group=dict(type='int'),
+        group=dict(type='str'),
```

Comment:
Will this type change break any backward compatibility?
Also, it's better to add the type of option in module documentation as well.

Reply:
@ganeshrn no this won't break backward compatibility.

Reply:
added in commit https://github.com/ansible/ansible/pull/54663/commits/929b4f078ab4cd105127ca513d18efde6a300b67